includes/class.llms.certificates.php

```diff
@@ -84,18 +84,16 @@ private function __construct() {
 	}
 
 	/**
-	 * Initialize Class
+	 * Initialize class.
 	 *
 	 * @since 1.0.0
 	 * @since 4.21.0 Define useful class properties used when exporting.
+	 * @since [version] Removed use of the deprecated LLMS_Certificate_User class.
 	 *
 	 * @return void
 	 */
 	public function init() {
 
-		include_once 'class.llms.certificate.php';
-		$this->certs['LLMS_Certificate_User'] = isset( $this->certs['LLMS_Certificate_User'] ) ? $this->certs['LLMS_Certificate_User'] : include_once 'certificates/class.llms.certificate.user.php';
```

Comment:
With this change `llms()->certificates()->certs['LLMS_Certificate_User']` will result in an error. For now this line should be left intact until `LLMS_Certificate_User` is fully removed.

Reply:
@thomasplevy,
Loading `class.llms.certificate.php` in line 96 is handled by the autoloader now.

Loading `certificates/class.llms.certificate.user.php` in line 97 will create a deprecation warning from [`LLMS_Certificate::_construct()`](https://github.com/gocodebox/lifterlms/blob/1931-refactor-engagement-sync/includes/class.llms.certificate.php#L142). Also, the [autoloader](https://github.com/gocodebox/lifterlms/blob/1931-refactor-engagement-sync/includes/class-llms-loader.php#L90) handles that class file.

Reply:
> Loading `class.llms.certificate.php` in line 96 is handled by the autoloader now.

I intended to only make a note on line 97, sorry.



> Loading `certificates/class.llms.certificate.user.php` in line 97 will create a deprecation warning from [`LLMS_Certificate::_construct()`](https://github.com/gocodebox/lifterlms/blob/1931-refactor-engagement-sync/includes/class.llms.certificate.php#L142). Also, the [autoloader](https://github.com/gocodebox/lifterlms/blob/1931-refactor-engagement-sync/includes/class-llms-loader.php#L90) handles that class file.

But if a dev accesses the property it'll create a fatal error because the the public class property isn't available.

Reply:
Now I understand why `LLMS_Certificate::__construct()` and `LLMS_Achievement::__construct()` didn't emit deprecation warnings. I will undo the `__construct()` deprecation warnings.